### DISCUSSION TITLE
Integrate Open-O1 into Open-WebUI

### DISCUSSION THREAD

**Discussion by zytoh0** [open-webui/open-webui#6007] [Ideas]
Integrate https://github.com/OpenSource-O1/Open-O1 into Open-WebUI

**Comment by rgaricano** [comment]
is a project, there isn't any model there, the model that are linked on running app is gpt4
\`\`\`
I'm OpenAI's GPT-4, the latest version of our advanced language generation model. I'm designed to handle a variety of tasks, from answering questions and translating languages to assisting with writing and more. How can I help you today?
\`\`\`
```
from V3 import call_gpt
...
    def ask(self, history, prompt):
        history = self.pop_history(history)
        logger.info(history)
        return call_gpt(history, prompt)
```
